electricity/src/lib.rs

```diff
@@ -1,53 +1,150 @@
 extern crate reqwest;
 
 use reqwest::Error;
+use tokio_cron_scheduler::{JobScheduler, Job};
+use scraper::{Html, Selector};
+use db::{DATABASE, create_table};
+use uuid::Uuid;
+use aws_sdk_dynamodb::{
+    types::{AttributeValue},
+    Client, Error as DynamoDBError
+};
+
+pub mod db;
 
 struct ShutDownParseData {
-    days_number: i8,
     page_url: String,
     city: String,
 }
 
-pub async fn console_lib() -> Result<(), Error> {
+struct TableRowData {
+    city: String,
+    region: String,
+    time: String,
+    street: String,
+    date: String,
+}
+
+async fn fetch_page(url: &str) -> Result<String, Error> {
+    let response = reqwest::get(url).await?;
+    let body = response.text().await?;
+
+    Ok(body)
+}
+
+pub async fn collect_data() -> () {
     let dates = vec![
         ShutDownParseData {
-            days_number: 0,
             page_url: String::from("https://elektrodistribucija.rs/planirana-iskljucenja-beograd/Dan_0_Iskljucenja.htm"),
             city: String::from("Belgrade")
         },
         ShutDownParseData {
-            days_number: 0,
             page_url: String::from("https://elektrodistribucija.rs/planirana-iskljucenja-beograd/Dan_1_Iskljucenja.htm"),
             city: String::from("Belgrade")
         },
         ShutDownParseData {
-            days_number: 0,
             page_url: String::from("https://elektrodistribucija.rs/planirana-iskljucenja-beograd/Dan_2_Iskljucenja.htm"),
             city: String::from("Belgrade")
         },
         ShutDownParseData {
-            days_number: 0,
             page_url: String::from("https://elektrodistribucija.rs/planirana-iskljucenja-beograd/Dan_3_Iskljucenja.htm"),
             city: String::from("Belgrade")
         },
-        
+        ShutDownParseData {
+            page_url: String::from("https://elektrodistribucija.rs/planirana-iskljucenja-srbija/NoviSad_Dan_0_Iskljucenja.htm"),
+            city: String::from("Novi Sad")
+        },
+        ShutDownParseData {
+            page_url: String::from("https://elektrodistribucija.rs/planirana-iskljucenja-srbija/NoviSad_Dan_1_Iskljucenja.htm"),
+            city: String::from("Novi Sad")
+        },
+        ShutDownParseData {
+            page_url: String::from("https://elektrodistribucija.rs/planirana-iskljucenja-srbija/NoviSad_Dan_2_Iskljucenja.htm"),
+            city: String::from("Novi Sad")
+        },
+        ShutDownParseData {
+            page_url: String::from("https://elektrodistribucija.rs/planirana-iskljucenja-srbija/NoviSad_Dan_3_Iskljucenja.htm"),
+            city: String::from("Novi Sad")
+        }
     ];
 
-    for date in dates {
-        let html = fetch_page(&date.page_url).await?;
+    let mut table_rows: Vec<TableRowData> = vec![];
+
+    for input in dates {
+        let html = fetch_page(&input.page_url).await.unwrap();
```

Comment:
I wish we could fetch pages concurrently, and parse them upon reception in a blocking thread.

I would rather extract the parsing logic from this function.